Tools/Scripts/libraries/webkitcorepy/webkitcorepy/tests/string_utils_unittest.py

```diff
@@ -67,6 +67,13 @@ def test_list(self):
         self.assertEqual(string_utils.join([]), 'Nothing')
         self.assertEqual(string_utils.join(['integer', 'string'], conjunction='or'), 'integer or string')
 
+    def test_split(self):
+        self.assertEqual(string_utils.split('integer, string and float'), ['integer', 'string', 'float'])
+        self.assertEqual(string_utils.split('integer and string'), ['integer', 'string'])
+        self.assertEqual(string_utils.split('integer'), ['integer'])
+        self.assertEqual(string_utils.split(''), [])
+        self.assertEqual(string_utils.split('integer or string', conjunction='or'), ['integer', 'string'])
```

Comment:
Should we add the edge case string_utils.split(' or ', conjunction='or') which should return []?

Reply:
Also add a test case for a string like "integer and float or boolean".

Reply:
Also add the test case 'interger or float and boolean'.

Reply:
also add a test case for `integer or float and boolean`.

Reply:
Also add another test case ...

Reply:
Test comment